msteams-platform/concepts/device-capabilities/browser-device-permissions.md

```diff
@@ -9,26 +9,36 @@ ms.topic: how-to
 # Device permissions for the browser
 
 > [!NOTE]
-> The change to how device permissions are handled in the browser is currently available in [public developer preview](../../resources/dev-preview/developer-preview-intro.md) only. 
-> This change will be generally available (GA) by January 21, 2022.
+> The new feature on how device permissions are handled in the browser are currently available in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
+> This new feature will be generally available (GA) by January 21, 2022.
 
-Applications that require device permissions - such as camera or microphone access - now require users to manually grant consent at a per app level in the web browser. Previously, the browser handled how these permissions were granted, but now these permissions will be handled in Microsoft Teams. This has implications on how you design your application if they require these permissions in the browser.
+Teams apps that require device permissions, such as camera or microphone access, now require users to manually grant permission at an app level in the web browser. 
 
-## Change in behavior
-If your application has declared that it needs device permissions in your [application manifest](native-device-permissions.md), then users will be shown an "app permissions" option where they can enable an app’s device permissions. The "app permissions" option can be found in personal apps, task module dialogs, and tabs in chats, channels or meetings.
+Previously, browser handled how to grant access permissions. Now these permissions are handled in Microsoft Teams. This has implications on how you design your application and if they require these permissions in the browser.
```

Comment:
Incorrect, please rephrase.

Reply:
The modified sentence (in green) is for the previous comments. Not applicable for the line no 18(in red)